make/libraries

```diff
@@ -33,6 +33,10 @@ $(CVODES)/lib/libsundials_nvecserial.a: $(SUNDIALS_NVECSERIAL)
 
 LIBCVODES = $(CVODES)/lib/libsundials_nvecserial.a $(CVODES)/lib/libsundials_cvodes.a
 
+STAN_CVODES_HEADERS := $(shell find stan -name *cvodes*.hpp)
```

Comment:
I'm happy to cross that bridge when we come to it, especially since this seems like something that only math library developers will encounter, right?